helix-term/src/ui/completion.rs

```diff
@@ -379,7 +398,7 @@ impl Completion {
         // > The returned completion item should have the documentation property filled in.
         // https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_completion
         let current_item = match self.popup.contents().selection() {
-            Some(item) if item.documentation.is_none() => item.clone(),
+            Some(item) if !item.resolved => item.clone(),
```

Comment:
🚀 I'm glad to see 25bbd13 for this case in particular: checking `documentation.is_none()` felt quite hacky

Reply:
(and was the cause of #6261 since the server didn't return any documentation and therefore caused continuous resolve requests)